src/ray/common/ray_config_def.h

```diff
@@ -98,6 +98,12 @@ RAY_CONFIG(bool, lineage_pinning_enabled, false)
 /// See also: https://github.com/ray-project/ray/issues/14182
 RAY_CONFIG(bool, preallocate_plasma_memory, false)
 
+/// The interval before the scheduling class capacity is increased. If set to
```

Comment:
Can you explain this a bit more? It's hard to understand what "scheduling class capacity" means without the context about worker processes, deadlock, etc.